### ISSUE TITLE
Synchronous getArchive

### ISSUE THREAD

**Issue by RangerMauve** [beakerbrowser/dat-node#4]
Currently it's really difficult to emulate the DatArchive API for [webrun](https://github.com/RangerMauve/node-web-loader) using this API because while I can await the archive to be loaded before invoking most of the methods, there's no way to do that with `archive.watch()` and other synchronous events.

**Comment by pfrazee** [comment]
This might not be easy to do with the current codebase

**Comment by RangerMauve** [comment]
Yeah... I kinda assumed that. For now I'm gonna try to change my codebase to use `DatArchive.load` instead of `new DatArchive` and am gonna try to encourage it in docs.
